test/e2e/frictionless/page-objects/frictionless.page.js

```diff
@@ -31,6 +31,8 @@ export class FrictionlessPage extends classes(DcGnavPage, PdfWidgetSection) {
       signUp: '[href*="https://auth.services.adobe.com"][href*="signup"]',
       extensionModal: '#chromeext, #edgeext',
       closeExtensionModal: '#chromeext .dialog-close, #edgeext .dialog-close',
+      caas: '[data-path$="/caas/default"]>>nth=0',
```

Comment:
CaaS are used in other pages too, not only Frictionless pages such as https://www.adobe.com/acrobat/resources/best-pdf-software.html